packages/test-base/src/androidAndroidTest/kotlin/io/realm/kotlin/test/shared/RealmDictionaryTests.kt

```diff
@@ -1113,6 +1190,311 @@ internal abstract class ManagedDictionaryTester<T>(
         }
     }
 
+    override fun values_addTrows() {
+        val dataSet = typeSafetyManager.dataSetToLoad
+
+        errorCatcher {
+            realm.writeBlocking {
+                val dictionary = typeSafetyManager.createContainerAndGetCollection(this)
+                val values = dictionary.values
+                assertFailsWithMessage<UnsupportedOperationException>("Adding values to a dictionary through 'dictionary.values' is not allowed") {
+                    values.add(dataSet[0].second)
+                }
+                assertFailsWithMessage<UnsupportedOperationException>("Adding values to a dictionary through 'dictionary.values' is not allowed") {
+                    values.addAll(listOf(dataSet[0].second))
+                }
+                assertTrue(dictionary.isEmpty())
```

Comment:
Aren't this tested in the cleanup method just below? The same seems to happen in all the other tests.